e2e/cypress/test_module/cypress/support/step_definitions/orders-filters.steps.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+
+import { Given, When, Then } from "@badeball/cypress-cucumber-preprocessor";
+import {
+  deliveryMethods,
+  orderDeliveryStatuses
+} from "@cypress-e2e/fixtures/constants";
+
+When("I can see orders filter button", () => {
+  cy.getById("filter-button").should("be.visible");
+});
+
+When("I click on the orders filter button", () => {
+  cy.getById("filter-button").click();
+});
+
+When("I should see the orders filter drawer", () => {
+  cy.getById("orders-filter-drawer").should("be.visible");
+});
+
+When("I click on the apply button", () => {
+  cy.getById("apply-filters-button").click();
+});
+
+Then("I should see the orders filter drawer closed", () => {
+  cy.getById("orders-filter-drawer").should("not.be.visible");
+});
+
+Given("I select order status {string}", (orderStatus: string) => {
+  cy.getById(`order-${orderDeliveryStatuses[orderStatus]}-status`).click();
+});
+
+When("I select creation date {string}", (creationDate: string) => {
+  cy.getById("creation-date-select").click();
+  cy.getById(`${creationDate}-date-select`).click();
+});
+
+When("I select delivery method {string}", (deliveryMethod: string) => {
+  cy.getById(`delivery-method-${deliveryMethods[deliveryMethod]}`).click();
+});
+
+When(
+  "I set price range from {string} to {string}",
+  (priceFrom: string, priceTo: string) => {
+    cy.getById("price-range-from").type(priceFrom);
+    cy.getById("price-range-to").type(priceTo);
+  }
+);
+
+When("I set is paid checkbox to {string}", (isPaid) => {
+  const isPaidCheckbox = cy.getById("is-paid-checkbox");
+  if (isPaid === "true") {
+    isPaidCheckbox.click();
+  } else if (isPaid === "false") {
+    isPaidCheckbox.dblclick();
+  }
+});
```

Comment:
I guess a simple `if` `else` will be enough
like `if (isPaid === "true") {} `
`else {
    isPaidCheckbox.uncheck()
  }`
  
  because we have only boolean statement

Reply:
Inside the `feature` file we have `"-"`, it's the case where our filter not applies. Also, about `uncheck`, it's not working with mui checkboxes, because it uses labels, not inputs. That why actually we need this checks in these ways

Reply:
> Inside the `feature` file we have `"-"`, it's the case where our filter not applies. Also, about `uncheck`, it's not working with mui checkboxes, because it uses labels, not inputs. That why actually we need this checks in these ways

ok. I just suggest to use `if else` instead `else if`
you have only boolean condition. there won't some additional `else if` checks. 
In your case `isPaid` just `true` or `false`